data/tutorials/gs_00_up_and_running.md

```diff
@@ -11,88 +11,166 @@ date: 2021-05-27T21:07:30-00:00
+
+If you are willing to set up OCaml on Windows, you might be interested in
+reading [OCaml on Windows](/docs/ocaml-on-windows) first.
+The following instructions work on Linux, BSD and macOS but also on Cygwin and
+WSL.
 
 ## Installing OCaml
 
-It's a straightfoward process to install OCaml. [Opam](https://opam.ocaml.org/) is OCaml's package manager, so we'll install it first. You will also use Opam when installing third-party OCaml libraries.
+The **platform installer** is currently in active development. It will
+automatically install Opam, OCaml and the development tools. [See below](#up-and-running-with-the-platform-installer)
+for the instruction.
+
+Please note that the installer is work in progress and might not work on your system.
+
+OCaml is available as a package in most linux distributions, however, it is
+often outdated. On the contrary, OCaml's package manager `opam` allows to
+easily switch from one version of OCaml to the other and much more. This is
+especially useful since different projects might require different versions of
+OCaml.
+
+So the best way to install `ocaml` is in fact by using `opam`, OCaml's package manager.
 
-Find the all installation instructions for both Unix-like systems and Windows below. If you're new to the CLI, the code blocks (in black) show the required commands (the text after # gives more information on the following commands). Type each command after the prompt $, although it's often represented by a %, >, or another symbol as well. Ensure you use the exact case and spacing shown, then hit return/enter at the end of every line.
+### Installing `opam`
 
-### Installation for Linux and macOS
+Alternatively, you can use the alpha-version platform installer to install Opam and the development tools, [see below](#up-and-running-with-the-platform-installer).
 
-In the cases below, `# Homebrew` indicates the command if you're using [Homebrew](https://brew.sh/) as the installer, and `# MacPorts` is if you're using [MacPorts](https://www.macports.org/) to install.
+[Opam](https://opam.ocaml.org/) is the package manager of OCaml. It introduces the concept of "switches", consisting of a compiler together with a set of packages (libraries and other files). Switches are used to have independent sets of dependencies in different projects.
+
+After having installed `opam`, you will need to initialise it, [see below](#initialize-opam).
+
+To install `opam`, you can [use your system package manager](https://opam.ocaml.org/doc/Install.html#Using-your-distribution-39-s-package-system), or download the [binary distribution](https://opam.ocaml.org/doc/Install.html#Binary-distribution). The details are available in the above links, but for convenience, we copy a few of them here:
```

Comment:
```suggestion
To install opam, you can [use your system package manager](https://opam.ocaml.org/doc/Install.html#Using-your-distribution-39-s-package-system), or download the [binary distribution](https://opam.ocaml.org/doc/Install.html#Binary-distribution). The details are available in the above links, but for convenience, we copy a few of them here:
```

I'd personally prefer to either capitalise opam or wrap it in monospace, but Thomas decided the formatting for lower case, no monospace as the official name. We're only using monospace when talking about the command `opam`